CHANGELOG.md

```diff
@@ -20,6 +20,12 @@ Version <dev> brings support for gleaning a Docker container id from cgroups v2
 
   For compatibility with Ruby 3.4 and to silence compatibility warnings present in Ruby 3.3, declare a dependency on the `base64` gem. The New Relic Ruby agent uses the native Ruby `base64` gem for Base 64 encoding/decoding. The agent is joined by Ruby on Rails ([rails/rails@3e52adf](https://github.com/rails/rails/commit/3e52adf28e90af490f7e3bdc4bcc85618a4e0867)) and others in making this change in preparation for Ruby 3.3/3.4.
 
+- **Fix: Stop sending duplicate log events for Rails 7.1 users**
+
+  Rails 7.1 introduced the public API [`ActiveSupport::BroadcastLogger`](https://api.rubyonrails.org/classes/ActiveSupport/BroadcastLogger.html). This logger replaces a private API, `ActiveSupport::Logger.broadcast`. In Rails versions below 7.1, the agent uses the `broadcast` method to stop duplicate logs from being recoded by broadcasted loggers. Now, we've updated the code to provide a similar duplication fix with the new `ActiveSupport::BroadcastLogger` class.
+
+  Previously, the agent prevented broadcasted loggers from recording duplicate log events by skipping instrumentation for broadcasted loggers, identified by calling `ActiveSupport::Logger.broadcast`. In Rails 7.1, this method has been refactored into a class. The agent now records log events for only the first logger found in an `ActiveSupport::BroadcastLogger.broadcasts` array.
```

Comment:
I'm only planning to keep one of these two paragraphs, @fallwith. Do you have a preference?

Reply:
If I can only have 1, the first one seems best. I link the link to BroadcastLogger documentation.